bundles/framework/timeseries/resources/css/timeseriesplayback.css

```diff
@@ -1,135 +1,197 @@
-    div.mapplugin.timeseries div.playback-button button.play {
-      width: 50px;
-      height: 40px;
-      background: #333438;
-      border: none;
-      margin: auto;
-      cursor: pointer; }
-      div.mapplugin.timeseries div.playback-button button.play:after {
-        content: '';
-        display: inline-block;
-        position: relative;
-        top: 1px;
-        left: 3px;
-        border-style: solid;
-        border-width: 10px 0 10px 20px;
-        border-color: transparent transparent transparent #ffde00; }
-    div.mapplugin.timeseries div.playback-button button.pause {
-      width: 50px;
-      height: 40px;
-      background: #333438;
-      position: relative;
-      display: block;
-      border: none;
-      cursor: pointer;
-      display: none; }
-      div.mapplugin.timeseries div.playback-button button.pause:before {
-        width: 8px;
-        height: 16px;
-        background: #ffde00;
-        position: absolute;
-        content: "";
-        top: 12.012012012px;
-        left: 16.6666666667px; }
-      div.mapplugin.timeseries div.playback-button button.pause:after {
-        width: 8px;
-        height: 16px;
-        background: #ffde00;
-        position: absolute;
-        content: "";
-        top: 12.012012012px;
-        right: 16.6666666667px; }
-  div.mapplugin.timeseries .oskari-timeslider {
-    margin-top: 10px;
-    width: 80%;
-    height: 40px;
-    background-color: #333438;
-    float: left; }
-    div.mapplugin.timeseries .oskari-timeslider .interval-line-highlight {
-      position: absolute;
-      top: 0;
-      z-index: 100001;
-      color: #ffde00;
-      height: 40px; }
-      div.mapplugin.timeseries .oskari-timeslider .interval-line-highlight .label {
-        font-size: 12px;
-        position: absolute;
-        top: 3px; }
-        div.mapplugin.timeseries .oskari-timeslider .interval-line-highlight .label.center {
-          left: -24px; }
-      div.mapplugin.timeseries .oskari-timeslider .interval-line-highlight .line {
-        width: 1px;
-        border-left: 1px solid #fff;
-        height: 12px;
-        position: absolute;
-        bottom: 0; }
-    div.mapplugin.timeseries .oskari-timeslider .interval-line {
-      position: absolute;
-      top: 0;
-      z-index: 100000;
-      height: 40px; }
-      div.mapplugin.timeseries .oskari-timeslider .interval-line .label {
-        font-size: 12px;
-        position: absolute;
-        top: 16px;
-        color: #fff; }
-        div.mapplugin.timeseries .oskari-timeslider .interval-line .label.center {
-          left: -14px; }
-      div.mapplugin.timeseries .oskari-timeslider .interval-line .line {
-        width: 1px;
-        border-left: 1px solid #fff;
-        height: 8px;
-        position: absolute;
-        bottom: 0; }
-  div.mapplugin.timeseries .oskari-timeslider-empty {
-    margin-top: 10px;
-    float: left;
-    width: 10px;
-    height: 40px; }
+/* override these to theme component */
```

Comment:
Nice thinking with the theming although the scss needs to be removed because of this.